doc/whats-new.rst

```diff
@@ -10,18 +10,33 @@ v0.6.0 (unreleased)
 
 New Features
 ~~~~~~~~~~~~
-- :py:meth:`~xgcm.grid.Grid.interp_like` enables users to interpolate arrays onto the grid positions of another array (:issue:`234` , :issue:`343`, :pull:`350`). By `Dianne Deauna <https://github.com/jdldeauna>`_ under the `SIParCS internship <https://www2.cisl.ucar.edu/siparcs-2021-projects#8>`_.
+- :py:meth:`~xgcm.grid.Grid.get_metric` refactored, and now incorporates :py:meth:`~xgcm.grid.Grid.interp_like` to allow for automatic interpolation of missing metrics from available values on surrounding positions (:pull:`345`, :pull:`354`).
+  By `Dianne Deauna <https://github.com/jdldeauna>`_.*
```

Comment:
Could you use the RST footnote syntax for the asterices? https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes